tests/spi/quay-imagepullsecret-usage.go

```diff
@@ -173,5 +178,50 @@ var _ = framework.SPISuiteDescribe(Label("spi-suite", "link-secret-sa"), func()
 			}, 1*time.Minute, 5*time.Second).Should(BeTrue(), "Pod not created successfully")
 		})
 
+		Describe("SVPI-408 - Check the secret that can be used with skopeo Tekton task to authorize a copy of one private Quay image to the second Quay image repository", Ordered, func() {
+			serviceAccountName := "tekton-task-service-account"
+
+			It("creates skopeo copy task", func() {
+				err := fw.AsKubeAdmin.TektonController.CreateSkopeoCopyTask(namespace)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("creates service account for the TaskRun referencing the docker config json secret", func() {
+				secrets := []corev1.ObjectReference{
+					{Name: SecretName},
+				}
+				_, err := fw.AsKubeAdmin.CommonController.CreateServiceAccount(serviceAccountName, namespace, secrets)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			var TaskRun *taskrunv1beta1.TaskRun
+			taskRunName := "skopeo-run"
+
+			It("creates taskrun", func() {
+				srcImageURL := fmt.Sprintf("docker://%s", TestQuayPrivateRepoURL)
+				destTag := fmt.Sprintf("spi-test-%s", strings.Replace(uuid.New().String(), "-", "", -1))
+				destImageURL := fmt.Sprintf("docker://%s:%s", QuayPrivateRepoURL, destTag)
+
+				TaskRun, err = fw.AsKubeAdmin.TektonController.CreateTaskRunCopy(taskRunName, namespace, serviceAccountName, srcImageURL, destImageURL)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("checks if taskrun is successful", func() {
+				Eventually(func() bool {
+					TaskRun, err = fw.AsKubeDeveloper.TektonController.GetTaskRun(taskRunName, namespace)
+
+					if err != nil {
+						return false
+					}
+
+					if len(TaskRun.Status.Conditions) == 0 {
+						return false
+					}
+
+					return (TaskRun.Status.Conditions[0].Status == "True")
```

Comment:
Would it make sense to check if `status` is `false` and `completionTime` is set? 
I believe that once `completionTime` is set (and status is not `true`), there is no way for the TaskRun to succeed. ([Docs](https://tekton.dev/docs/pipelines/taskruns/#monitoring-execution-status))
This could help fail the test early without waiting for timeout.

Reply:
@tnevrlka, thank you so much for your suggestion. I will change the test :)